packages/react/src/components/molecules/HeaderNav/main-nav.js

```diff
@@ -224,31 +227,21 @@ export const HeaderNavItem = React.memo(({
 
   return(
     <li ref={itemRef} role="none" className={classes} tabIndex="-1">
-      {isCovid ? (
-        <a
-          role="menuitem"
-          href={href}
-          className="ma__main-nav__top-link cv-alternate-style"
-          tabIndex="0"
-        >
-          {text}
-        </a>
-      ) : (
-        <button ref={buttonRef} type="button" role="menuitem" id={`button${index}`} className="ma__main-nav__top-link" aria-haspopup="true" tabIndex="0" aria-expanded={buttonExpanded}>
-          <span className="visually-hidden show-label">Show the sub topics of </span>
-          {text}
-        </button>
-      )}
-      {hasSubNav && (
-        <div ref={contentRef} className={contentClasses}>
-          <ul id={id || `menu${index}`} role="menu" aria-labelledby={`button${index}`} className="ma__main-nav__container">
-            { subNav.map((item, itemIndex) => (
+      {hasSubNav ? (
+        <>
+          <button ref={buttonRef} type="button" role="menuitem" id={`button${index}`} className="ma__main-nav__top-link" aria-haspopup="true" tabIndex="0" aria-expanded={buttonExpanded}>
+            <span className="visually-hidden show-label">Show the sub topics of </span>
+            {text}
+          </button>
+          <div ref={contentRef} className={contentClasses}>
+            <ul id={id || `menu${index}`} role="menu" aria-labelledby={`button${index}`} className="ma__main-nav__container">
+              { subNav.map((item, itemIndex) => (
               // eslint-disable-next-line react/no-array-index-key
-              <li key={`main-nav-subitem--${index}-${itemIndex}`} role="none" className="ma__main-nav__subitem">
-                <a aria-expanded={buttonExpanded} onClick={onButtonLinkClick} role="menuitem" href={item.href} className="ma__main-nav__link">{item.text}</a>
-              </li>
-            ))}
-            { href && (
+                <li key={`main-nav-subitem--${index}-${itemIndex}`} role="none" className="ma__main-nav__subitem">
```

Comment:
I don't _think_ we want to blank out the `role` on `<li>`s in this context. Is there a reason to do this I don't know about?